src/applications/yellow-ribbon/components/SearchResult/index.jsx

```diff
@@ -69,8 +76,25 @@ const deriveDegreeLevelLabel = (school = {}) => {
 
 const deriveProgramLabel = () => 'Unknown';
 
-const SearchResult = ({ school }) => (
-  <div className="medium-screen:vads-l-col vads-l-col vads-u-background-color--gray-light-alt vads-u-margin-bottom--2 vads-u-padding-x--3 vads-u-padding-y--2">
+export const SearchResult = ({
+  addSchoolToCompare,
+  removeSchoolFromCompare,
+  school,
+  schoolIDs,
+}) => (
+  <div
+    className={classNames({
+      'medium-screen:vads-l-col': true,
+      'vads-l-col': true,
+      'vads-u-background-color--gray-light-alt': true,
+      'vads-u-border--3px': true,
+      'vads-u-border-color--primary': includes(schoolIDs, school?.id),
+      'vads-u-border-color--transparent': !includes(schoolIDs, school?.id),
+      'vads-u-margin-bottom--1': true,
+      'vads-u-padding-x--3': true,
+      'vads-u-padding-y--2': true,
+    })}
```

Comment:
You could move out the `true` properties here into other args, like -

```js
className={classNames(
    'medium-screen:vads-l-col', 
    'vads-l-col', 
    'vads-u-margin-bottom--1', 
    'vads-u-padding-x--3',
    'vads-u-padding-y--2', 
   'vads-u-background-color--gray-light-alt',
   'vads-u-border--3px', {
      'vads-u-border-color--primary': includes(schoolIDs, school?.id),
      'vads-u-border-color--transparent': !includes(schoolIDs, school?.id),
    })}
```

Reply:
Beautiful! Thanks!! :)